src/main.tsx

```diff
@@ -1,4 +1,23 @@
-import parade from './index';
-import config from './example/config';
+import parade, { Config, WidgetConfigMap } from './index';
+import baseConfig from './example/config';
+import { dirname, sep } from 'path';
+
+const getLocalConfig = (require as any).context('./', true, /\.example\.ts(x)?$/);
+
+const widgetConfig = getLocalConfig.keys().reduce((map: WidgetConfigMap, id: string) => {
+	const widgetKey = dirname(id)
+		.split(sep)
+		.pop();
+	map[widgetKey || id] = getLocalConfig(id).default;
+	return map;
+}, {});
+
+const config: Config = {
+	...baseConfig,
+	widgets: {
+		...baseConfig.widgets,
+		...widgetConfig
+	}
+};
```

Comment:
Note that mixing both global widget configuration and local `.example.ts` configuration is possible for the sake of as much compatibility as possible, with localized configurations taking precedence.